examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -793,6 +876,7 @@ def main(args):
         tokenizer=tokenizer,
         size=args.resolution,
         center_crop=args.center_crop,
+        encoder_hidden_states=pre_computed_encoder_hidden_states,
```

Comment:
Didn't see `encoder_hidden_states` yet in the `DreamBoothDataset` class:

https://github.com/huggingface/diffusers/blob/0ffac97933d8ca1487d4ae9c374504a0979f0c5d/examples/dreambooth/train_dreambooth_lora.py#L421-L431

Reply:
Hmm, I think you're linking the current head commit, not in the PR?

Reply:
Ah, sorry disregard that. 

But can we also do the same for the [class prompt](https://github.com/williamberman/diffusers/blob/b86ddc43afc5f2c22cbdf550d8836c9563521f5c/examples/dreambooth/train_dreambooth_lora.py#L511)?

Reply:
https://github.com/huggingface/diffusers/pull/3360/commits/4042274a1a3df6c587fc160eacb7f4f8058a4c8a good point, done!